documentation/operators/src/nodes/gateway-setup.md

```diff
@@ -56,39 +60,110 @@ To check available configuration options use:
 ```
 ~~~
 
-The following command returns a gateway on your current IP with the `id` of `supergateway`:
+The following command returns a gateway on your current IP with the `<ID>` of `supergateway`:
 
 ```
-./nym-gateway init --id supergateway --host $(curl ifconfig.me) --wallet-address n1eufxdlgt0puwrwptgjfqne8pj4nhy2u5ft62uq
+./nym-gateway init --id supergateway --host $(curl ifconfig.me)
 ```
 
 ~~~admonish example collapsible=true title="Console output"
 ```
-<!-- cmdrun ../../../../target/release/nym-gateway init --id supergateway --host $(curl ifconfig.me) --wallet-address n1eufxdlgt0puwrwptgjfqne8pj4nhy2u5ft62uq -->
+<!-- cmdrun ../../../../target/release/nym-gateway init --id supergateway --host $(curl ifconfig.me) -->
 ```
 ~~~
 
 The `$(curl ifconfig.me)` command above returns your IP automatically using an external service. Alternatively, you can enter your IP manually if you wish. If you do this, remember to enter your IP **without** any port information.
 
+
+#### Initialising gateway with network requester
+
+As some of the [Project Smoosh](../faq/smoosh-faq.md) changes getting implemented, the gateways now can work also as a network requesters at the same time. Such combination creates an exit gateway node, needed for new more open setup. 
+
+An operator can initialise the gateway and network requester together by running:
```

Comment:
Pick term and stick to it (see comments below) - we've introduced the post-smoosh gateway as an 'exit gateway', lets stick to this. Its clear (people will think of tor exit node) plus consistent

Reply:
i go with exit gateway everywhere to entrench it.